test/integration/file-uploads/request-body.test.js

```diff
@@ -645,4 +645,81 @@ describe('file upload in request body', function () {
                 .to.equal('Binary file load error: file resolver interface mismatch');
         });
     });
+
+    (IS_BROWSER ? describe.skip : describe)('large file upload in request body', function () {
+        // eslint-disable-next-line mocha/no-sibling-hooks
+        before(function (done) {
+            this.run({
+                fileResolver: {
```

Comment:
add a comment explaining why this custom file resolver needed.

Reply:
Fixed